dss/api/bundles/__init__.py

```diff
@@ -130,13 +130,13 @@ def get(
     if link is None:
```

Comment:
on line 188 please remove the indent to allow `response.headers['X-OpenAPI-Paginated-Content-Key'] = 'bundles'` to be applied to all responses

Reply:
Thanks for catching that!